src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -58,6 +58,25 @@ interface AuctionResultsProps {
   artist: ArtistAuctionResults_artist$data
 }
 
+interface AuctionMetaTagsProps {
+  meta: {
+    title: string
+    description: string
+  }
+}
```

Comment:
We don't manually type props like this. You'd want to use a fragment container. I would probably just inline this in the parent component; there's nothing really here.